### PR TITLE
Add Apache 2.0 license

### PR DESCRIPTION
**Pull request by omichel**
👍

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (DavidMansolino) [inline LICENSE]: Shouldn't we add date a name here?

**Comment by omichel** [comment]
👍

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,13 @@
+   Copyright 2019 Cyberbotics Ltd.
+   
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use the file in this repository except in compliance with the License.
```
Comment (DavidMansolino) [inline LICENSE]:
```suggestion
   you may not use the files in this repository except in compliance with the License.
```

**Comment by omichel** [comment]
👍

**Review by DavidMansolino — APPROVED** [review]
Thank you

**Pull request merged**